resolwe_bio/docker_images/chipseq/Dockerfile

```diff
@@ -0,0 +1,72 @@
+FROM docker.io/resolwe/base:ubuntu-14.04
+
+LABEL MAINTAINER Resolwe-bio authors https://github.com/genialis/resolwe-bio
+
+RUN apt-get update && \
+    apt-get install -y software-properties-common python python-dev python-pip && \
```

Comment:
Use `apt-get -y install --no-install-recommends software-properties-common` here. It reduces the number of installed packages from 40 to 33.

Why would you want to install `python`, `python-dev` and `python-pip` separately here instead of where you install all packages later?